src/askui_runner/modules/agents/models.py

```diff
@@ -0,0 +1,38 @@
+import os
```

Comment:
Would rename the module from `models` to `config` to be more specific about what this is.